src/stylesheets/style.css

```diff
@@ -396,9 +399,19 @@ div {
 #fcc_foldout_toggler {
   position: absolute;
   top: 20px;
-  left: 20px;
+  left: 15px;
   z-index: 99998;
 }
+#fcc_foldout_toggler_background {
```

Comment:
Are these style changes important to the functionality of keeping the UI closed? If not, it would have been better to put it in a separate PR, but I'm not going to make a fuss about it. Just a reminder to try to keep PRs small and focused on one change so we can more easily reverse each one without the reversing the other, if there are bugs.

Reply:
The changes were in reference to the comment above by @QuincyLarson re: making it obvious how to un-collapse the menu. This is a bolder toggler with a faint background.